argoverse/utils/interpolate.py

```diff
@@ -181,7 +180,7 @@ def interp_arc(t: int, px: np.ndarray, py: np.ndarray) -> np.ndarray:
     eq_spaced_points = np.linspace(0, 1, t)
 
     # how many points will be uniformly interpolated?
-    nt = eq_spaced_points.size
```

Comment:
maybe this can go above in the inline comment, like
```python
# equally spaced in arclength -- the number of points that will be uniformly interpolated
eq_spaced_points = np.linspace(0, 1, t)
```

Reply:
👍